src/Snapshooter.Xunit.Tests/__snapshots__/SnapshotTests.Match_FactMatchSnapshotWithCrLfStringInObject_SuccessfulMatch.snap

```diff
@@ -1,7 +1,7 @@
-{
+﻿{
   "Id": "c78c698f-9ee5-4b4b-9a0e-ef729b1f8ec8",
   "Firstname": "Mark",
-  "Lastname": "Your\nName\nAtHome",
+  "Lastname": "Your\r\nName\nAt\rHome",
```

Comment:
Maybe I'm missing something here but wasn't the point of this PR that only `\n` would be used now?

Reply:
you are right, we should fix this in a way that we only have `\n`.

Reply:
@maracuja-juice in your case the the line feeds are serialized, those should be fine. We had problems with the non-serialized ones. We should have a look if we can normalize these once two. 

@nscheibe we should open an issue for this one to track and describe the pitfalls here.

Reply:
Just to explain the problem we had.... In the previous versions we deleted all '\r', because of compare troubles between unix '\n' and windows '\r\n' file line endings... 
Now, we had some path strings in the snapshot with '\r' (example: ..\company\repos\hotchocolate), which were deleted (\repos --> epos) and the comparison failed.
Therefore we agreed on the following, we only delete '\r' if it is directly followed by a '\n' --> hence all \r\n will be replaced by \n .
Maybe my title and description were not really exact... sorry...

@michaelstaib lets discuss a proper solution in the next coffee break :)

Reply:
Okay that makes sense.
Keep us updated about the proposals for a proper solution.